skimage/io/tests/test_pil.py

```diff
@@ -97,6 +97,17 @@ def test_imread_truncated_jpg():
                   os.path.join(data_dir, 'truncated.jpg'))
 
 
+def test_jpg_quality_arg():
+    chessboard = np.load(os.path.join(data_dir, 'chessboard_GRAY_U8.npy'))
+    with NamedTemporaryFile(suffix='.jpg') as jpg:
```

Comment:
And if delete is set to False, we also have to manually unlink.

On Mon, Mar 30, 2015 at 7:29 PM, Steven Silvester <notifications@github.com>
wrote:

> In skimage/io/tests/test_pil.py
> <https://github.com/scikit-image/scikit-image/pull/1460#discussion_r27448896>
> :
>
> > @@ -97,6 +97,17 @@ def test_imread_truncated_jpg():
> >                    os.path.join(data_dir, 'truncated.jpg'))
> >
> >
> > +def test_jpg_quality_arg():
> > +    chessboard = np.load(os.path.join(data_dir, 'chessboard_GRAY_U8.npy'))
> > +    with NamedTemporaryFile(suffix='.jpg') as jpg:
>
> Based on the docs
> <https://docs.python.org/2/library/tempfile.html#tempfile.NamedTemporaryFile>,
> I think you'll need to close the temporary file before calling imsave on
> Windows.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-image/scikit-image/pull/1460/files#r27448896>.
>